artifactory.py

```diff
@@ -1962,7 +2058,20 @@ def del_properties(self, properties, recursive=None):
         recursive  - on folders property attachment is recursive by default. It is
                      possible to force recursive behavior.
         """
-        return self._accessor.del_properties(self, properties, recursive)
+        properties_to_remove = dict.fromkeys(properties, None)
+        # Uses update properties since it can consume JSON as input and removes URL limit
+        self.update_properties(properties_to_remove, recursive=recursive)
+
+    def update_properties(self, updated_props_json, recursive=False):
```

Comment:
`updated_props_json` is it really json? Like string object with JSON inside? Or just `properties`

Reply:
>specified (dict) in docstrings
python suggests good instrument for it - type hints! It works in "comment" mode for all python's versions

We should look at them instead, but for now - it's enough, TY!